staging/src/k8s.io/apiserver/pkg/admission/audit.go

```diff
@@ -27,7 +28,9 @@ import (
 // auditHandler logs annotations set by other admission handlers
 type auditHandler struct {
 	Interface
-	ae *auditinternal.Event
+	// to protect the 'Annotations' map of the audit event from concurrent writes
+	mutex sync.Mutex
```

Comment:
can you please also add a TODO here:
`TODO:  move the lock near the `Annotations` field of the of the audit events so it is always protected from concurrent access.`